### ISSUE TITLE
Instead of writing new .gz files, source files are overwritten (starting from 0.10)

### ISSUE THREAD

**Issue by givankin** [gruntjs/grunt-contrib-compress#132]
Hi!

I use the example conf. from README.md:

```js
// gzip assets 1-to-1 for production
compress: {
  main: {
    options: {
      mode: 'gzip'
    },
    expand: true,
    cwd: 'assets/',
    src: ['**/*'],
    dest: 'public/'
  }
}
```

However, the gzipped files in `dest` have the same filenames as source files, without the .gz extension.

Furthermore, when I modify the configuration so that .gz files are put side-by-side with source files
```js
compress: {
      main: {
        options: {
          mode: 'gzip'
        },
        expand: true,
        src: ['assets/**/*']
      }
    },
```

then all the source files are replaced with broken gzip files (filenames do not change), each 20 bytes long (`‹              `).

Before 0.10 both configurations run as expected.

**Comment by XhmikosR** [comment]
Please someone make a PR for this to add the info in README.

**Comment by philippsimon** [comment]
i just fixed issue #61 and made apull request which will fix also this issue (they are actually the same)
